office/task/Portfile

```diff
@@ -4,9 +4,9 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cmake 1.0
```

Comment:
can you please update this to `cmake 1.1`? The line `cmake.out_of_source yes` later in the Portfile can then be removed.

Reply:
done.